g/go-oidc/go-oidc_ubi8_5.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : go-oidc
+# Version       : v2.2.1
+# Source repo   : https://github.com/coreos/go-oidc.git
+# Tested on     : UBI 8.5
+# Language      : go
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation {maintainer}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=go-oidc
+PACKAGE_VERSION=v2.2.1
+PACKAGE_URL=https://github.com/coreos/go-oidc.git
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
```

Comment:
Do we need these packages : `nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses` ?